tests/neg/i17183.check

```diff
@@ -1,14 +1,18 @@
 -- [E081] Type Error: tests/neg/i17183.scala:11:24 ---------------------------------------------------------------------
 11 |def test = Context(f = (_, _) => ???) // error // error
    |                        ^
-   |                        Missing parameter type
+   |                        Could not infer type for parameter _$1 of anonymous function
```

Comment:
I wonder if it would be possible to report
```
 Could not infer type for the 1st parameter of an anonymous function
```

Reply:
I think it would be possible, but I'm not sure it's desirable, I think here we need to show some internals

Reply:
The internal names of `_` and other unamed variables are only useful to us. Never to the user as they cannot use them in the source.

Reply:
I agree, but we need to show the desugarring anyways, as the issue might be that the user missunderstood how `_`works, so we need names for these identifiers anyways